x-pack/docs/en/security/securing-communications/tls-http.asciidoc

```diff
@@ -1,42 +1,58 @@
 [role="xpack"]
 [[tls-http]]
-==== Encrypting HTTP Client communications
+==== Encrypting HTTP client communications
 
 When {security-features} are enabled, you can optionally use TLS to ensure that
 communication between HTTP clients and the cluster is encrypted.
 
 NOTE: Enabling TLS on the HTTP layer is strongly recommended but is not required.
 If you enable TLS on the HTTP layer in {es}, then you might need to make
-configuration changes in other parts of the Elastic Stack and in any {es}
-clients that you use.
+configuration changes in other parts of the {stack} and in any {es} clients that
+you use.
 
 . If you have not done so already, <<node-certificates,generate node certificates>>.
++
+--
+In particular, you need the files that are generated by the following command:
+
+[source,shell]
+----------------------------------------------------------
+bin/elasticsearch-certutil http
+----------------------------------------------------------
+
+This command generates a zip file that contains certificates and keys for use in {es} and {kib}. Each folder contains a readme that explains how to use the files.
+--
+
+. Verify that you've copied the output files to the appropriate locations, as
+specified in the readme files. 
++
+--
+For example, copy the `http.p12` file from the `elasticsearch` folder into a 
+directory within the {es} configuration directory on each node. If you chose to 
+generate one certificate per node, copy the appropriate `http.p12` file to each
+node. If you want to use {kib} to access this cluster, copy the
+`elasticsearch-ca.pem` file from the `kibana` folder into the {kib}
+configuration directory.
+--
 
 . Enable TLS and specify the information required to access the node’s
-certificate.
+certificate. Follow the instructions in the readme in the `elasticsearch` folder
+of the zip file. For example:
 
-** If the certificate is in PKCS#12 format, add the following information to the
-`elasticsearch.yml` file on each node:
+.. Update the `elasticsearch.yml` file on each node with the location of the
+certificates.
 +
 --
+If the certificates are in PKCS#12 format:
+
 [source, yaml]
 --------------------------------------------------
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.keystore.path: certs/elastic-certificates.p12 <1>
-xpack.security.http.ssl.truststore.path: certs/elastic-certificates.p12 <2>
+xpack.security.http.ssl.keystore.path: "http.p12"
```

Comment:
I think that since we talked about using a certs folder in previous examples, it should be used here too:
```suggestion
xpack.security.http.ssl.keystore.path: certs/http.p12
```

Reply:
Maybe we should drop the `certs/` part entirely? 
We don't use it in the PEM examples (and that uses more files)